Packs/CTIX/ReleaseNotes/2_2_7.md

```diff
@@ -0,0 +1,4 @@
+#### Integrations
+##### CTIX v3
+- Updated `expires` value.
+- Made arguments case insensitive.
```

Comment:
```suggestion
-  Documentation and metadata improvements.
```
since the changes has no visible impact on the user, its better to use ` Documentation and metadata improvements.` line instead of both lines

Reply:
@merit-maita Added the changes.